x-pack/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -220,6 +220,17 @@ export class HeadlessChromiumDriverFactory {
       })
     );
 
+    const uncaughtExceptionPageError$ = Rx.fromEvent<Error>(page, 'pageerror').pipe(
+      map((err) => {
+        logger.error(
+          i18n.translate('xpack.reporting.browsers.chromium.pageErrorDetected', {
+            defaultMessage: `Reporting encountered an error on the page: {err}`,
```

Comment:
Should this log line be `warning` since the error is ignored? And/or should the content mention that we are ignoring the error?

Catching and logging the error makes this similar to the `pageRequestFailed$` observable. 

I am in favor of updating the content on line 228 to remove the `This error will be ignored` text. That has confused users in times when the report eventually does fail. But I do think this log level should be `warning` since we're not `throw`ing the error after logging it.